data/json/items/comestibles/dairy.json

```diff
@@ -107,6 +107,26 @@
     "vitamins": [ [ "vitA", 7 ] ],
     "fun": -1
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "raw_butter",
+    "name": "raw butter",
+    "name_plural": "raw butter",
+    "weight": "463 g",
```

Comment:
This will end up being **way** too dense because this is the weight *per item*, while `"volume"` is the volume of a single *unit* of said items, the size of which is defined by `"charges"` (or `"stack_size"` if present).

Reply:
OK I moved weight back to 14 grams which is the serving size and 33 charges is the number of servings in that volume size when I was googling.